src/views/setup/Home.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <div id="home">
+    <div class="centerize" v-if="!hasPrerequisites">
+      <p>
+        Before using this extension,
+        <a href="https://git-scm.com/downloads">Git</a>
```

Comment:
Link on hover is not readable

Reply:
I am getting no buttons to proceed or take action

![Screen Shot 2020-10-14 at 12 07 43 PM](https://user-images.githubusercontent.com/25585652/95952513-e3918900-0e15-11eb-8464-2a44e113b960.png)